articles/ndes-scep-proxy.md

```diff
@@ -0,0 +1,146 @@
+# Configuring and using NDES SCEP proxy
```

Comment:
```suggestion
#  Connect end users to Wi-Fi with Simple Certificate Enrollment Protocol (SCEP).                  
```

I think let's make frame this guide around the intended use case rather than the "how" (NDES SCEP proxy)